src/css/clustermanagement.css

```diff
@@ -418,6 +418,7 @@ div.nodetooltip {
   border: 1px solid;
   min-height:8rem;
   max-height: 18rem;
+  overflow: auto;
```

Comment:
This might cause the list to jump a bit if/when the scrollbar appears. Did you consider setting `averflow-y: scroll` and have the scrollbar always visible?

Reply:
This is the reason I am changing it.
It gets from `pre-scrollable` `overfloy-y: scroll` and constantly shows the scrollbar.
The list is not very dynamic so I don't think there will be a problem with jumping.